course_inventory/inventory.py

```diff
@@ -177,6 +177,10 @@ def gather_course_data_from_api(account_id: int, term_ids: Sequence[int]) -> pd.
 
     course_df = pd.DataFrame(course_dicts_with_students)
     course_df = course_df.drop(['total_students'], axis='columns')
+    orig_course_count = len(course_df)
+    course_df = course_df.drop_duplicates(subset=['canvas_id'], keep='last')
```

Comment:
Does `last` seem right in this case? Not sure it matters too much...

Reply:
either last or first should be fine.

I wonder whether the duplicate course could exist in course_dicts (line 167) already? Should the check duplicate id be done at that level?

Reply:
I mean, either way the duplicates would be dropped, right? It's easier to deal with duplicates with `pandas`; I suppose I could create the dataframe earlier and do the filtering with `pandas` also. Were you concerned about something in particular?

Reply:
Yes, the duplicates would be dropped in the end anyway. Please go ahead with the current impl.